api/src/opentrons/protocol_engine/actions/actions.py

```diff
@@ -134,6 +135,14 @@ class AddLabwareDefinitionAction:
     definition: LabwareDefinition
 
 
+@dataclass(frozen=True)
+class AddLiquidAction:
+    """Add a liquid, to apply to subsequent `LoadLiquid`s."""
+
+    liquid: Liquid
```

Comment:
It might be better to recreate the parameters rather than tie ProtocolEngine internals to the public JSONv6 schema in this way

```suggestion
    display_name: str
    description: str
    display_color: Optional[str]
```